### ISSUE TITLE
Allow sending of email to related contacts of a case(AOP/AOW)

### ISSUE THREAD

**Issue by willrennie** [salesagility/SuiteCRM#136]
Ability to create a workflow that triggers to send an email to any/all contacts related to a case within a specific date period. This should be possible using repeated runs/any change.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Allow sending of email to related contacts of a case(AOP/AOW)](https://trello.com/c/oVrSe4p1/161-allow-sending-of-email-to-related-contacts-of-a-case-aop-aow)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
